tech team practices update (#14)
2i2c-org/team-compass: tech-team-practices.md

> # Staff sync practices

Reviewer: ```suggestion
# Tech team internal coordination process
```

We should communicate that:

1. This is for the tech team
2. Once we have volunteers, will they also participate?
3. The goal of this is to co-ordinate between the teams, not to 'report up'.

I'm not sold on my suggestion either, would love to iterate.

**Per-project to-dos and conversation**: Generally speaking specific projects will have their own place for team conversation, to-do items, and strategy. This is generally *not* in the 2i2c handbook, but there should always be links and references in the handbook that points somebody in the right direction.

**Discussion around team practices**: For discussion around strategy that doesn't have a natural place elsewhere, we use `handbook` issues and PRs. If an issue, it should be actionable (and "closeable"), generally in a way that results in a change to the handbook. If a PR, it should be a proposal to change the handbook and thus update some strategic information in there.

## Bi-weekly team syncs

Every two weeks we synchronize our team goals and practices. Here's how to conduct a team sync.

> 1. **Create an issue in `handbook/`**. This is where we'll keep track of the "to-do" for the team sync.

Reviewer: We could make this an issue template. Not a blocker for this though.

2. **Clean up [the HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw?both) for the sync**. We'll use [this HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw?both) to run the team sync. At the top is a set of questions to answer, and we can We'll initially use the HackMD to give responses. You can create it from the following template:

> 3. **Team members give their responses**. You can copy/paste the template, and then give your responses below. You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful. Just remember that others depend on this information to know what one another is up to!

Reviewer: ```suggestion
3. **Team members give their responses**. You can copy/paste the template, and then give your responses below. You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful. 
```

The extra sentence seemed a bit too much 'stick' :)

> 4. **We'll leave the HackMD open for 48hrs**. That should give enough time for those of us across many time zones to response.

Reviewer: ```suggestion
4. **We'll leave the HackMD open for 48hrs**, not counting weekends or holidays. That should give enough time for those of us across many time zones to respond.
```

Explicitly say that the 48h doesn't count holidays & weekends. 'Holidays' across countries is complicated but we can probably figure that out if it becomes a problem.
